client/Lykke.Service.ClientSearch.Client/Lykke.Service.ClientSearch.Client.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Version>1.0.1</Version>
+    <Authors>Lykke</Authors>
+    <Company>Lykke</Company>
+    <Product>Lykke.Service.ClientSearch</Product>
+    <Description>Client for Lykke.Service.ClientSearch API</Description>
+    <PackageLicenseUrl>https://github.com/LykkeCity/Lykke.Service.ClientSearch/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/LykkeCity/Lykke.Service.ClientSearch</PackageProjectUrl>
+    <PackageIconUrl>https://avatars3.githubusercontent.com/u/14153330?v=3&amp;s=200</PackageIconUrl>
+    <RepositoryUrl>https://github.com/LykkeCity/Lykke.Service.LykkeService.git</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageTags>Lykke</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.0" />
+    <PackageReference Include="Lykke.Common" Version="3.1.0" />
+    <PackageReference Include="Flurl.Http" Version="1.2.0" />
+  </ItemGroup>

```

Comment:
Enable xml documentation inclusion for debug and release configurations

Reply:
added